src/fonts.cpp

```diff
@@ -131,6 +131,10 @@ _pack_sdf_glyphs(
         stbtt_GetCodepointHMetrics(
             &font_info, sdf.codepoint, &horizontal_advance, &left_side_bearing);
 
+        if (rect.w == 0 && rect.h == 0){
+            rect.w = horizontal_advance * sdf.scale / 2;
+            rect.h = horizontal_advance * sdf.scale / 2;
+        }
```

Comment:
This will not behave well - artificially increasing glyph's rectangle will increase portion of rendered baked SDF texture (colliding with adjacent glyphs) - which will cause on-screen artifacts during rendering.

Reply:
Why would a baked portion of a whitespace glyph in ttf generate any visual artifacts? horizontal_advance comes from the font and it guarantees that the baked portion of the whitespace glyph is correct, because it is always SMALLER than w and h of the actual glyph in the ttf. 

Maciek made a bug here, but a different one. rect.h value should not be based off horizontal_advance, but be hardcoded to a very small, non-zero value e.g. rect.h = 0.1 representing glyph height of " " character. Other than that everything is correct, there should be no on-screen artifacts.

Reply:
In this piece of code `rect` represents an already packed SDF glyph (packed in spritesheet). If glyph was packed with width and height 0, 0 it's almost guaranteed that other glyph is packed right next to it. Increasing width and height will cause this space (or other whitespace char) to render part of this adjacent glyph. It also seems wasteful to render whitespace character knowing it will not produce any meaningful pixels.